### ISSUE TITLE
Have a BUG in scroll.js

### ISSUE THREAD

**Issue by vagra** [docsifyjs/docsify#1127]
#### Steps to reproduce

have a BUG in scroll.js 

this BUG cause: 

if path and id include chinese character, it can't find some result from the nav[].

#### What is current behaviour

```
export function scrollIntoView(path, id) {
  if (!id) {
    return;
  }
  const topMargin = config().topMargin;
  const section = dom.find('#' + id);
  section && scrollTo(section, topMargin);

  const li = nav[getNavKey(path, id)];
```

#### What is the expected behaviour

last line shoule changed to:
```
  const li = nav[getNavKey(decodeURIComponent(path), decodeURIComponent(id))];
```

#### Other relevant information

- [ ] Bug does still occur when all/other plugins are disabled?

- Your OS: 
- Node.js version: 
- npm/yarn version: 
- Browser version: 
- Docsify version: 
- Docsify plugins: 

#### Please create a reproducible sandbox 

[![Edit 307qqv236](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/307qqv236)

**Comment by vagra** [comment]
when the .md file name include chinese character:

```
console.log(path + "\t" + id);
```
result:
```
/docs/%E6%A5%9E%E4%B8%A5%E7%BB%8F%EF%BC%88%E4%BE%9D%E5%9C%86%E7%91%9B%E6%B3%95%E5%B8%88%E8%AE%B2%E4%B9%89%EF%BC%89	乙一、序分
```

```
console.log(decodeURIComponent(path) + "\t" + decodeURIComponent(id));
```
result:
```
/docs/楞严经（依圆瑛法师讲义）	乙一、序分
```

**Comment by vagra** [comment]
this problem cause: 
if .md file include chinese characters,  when click at the heading in main content, the sidebar links can't heilight and sync scroll.

because `const li = nav[getNavKey(path, id)]` have not result.

**Comment by anikethsaha** [comment]
we need to check whether this change will cause side effects or not. 
feel free to investigate and submit a fix.
Just a note that we don't have a strong test so we should check manually whether other things are breaking cause of this or not.

**Comment by anikethsaha** [comment]
fixed at #1146
